docs/about.rst

```diff
@@ -88,3 +88,5 @@ Contributors
 * Tom Barclay
 * Rollin C. Thomas
 * Danny Goldstein
+
+ LocalWords:  sncosmo
```

Comment:
Did you mean to add this line? Otherwise this PR looks good; thanks! (Oh, and feel free to add yourself to the contributors list while you're at it... every commit counts.)